Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <sstream>
+#include <string>
+
+
+#include "Person.h"
+
+
+Person::Person(){
+  age_ = new int (0);
+}
+
+Person::~Person(){
+    delete age_;
+}
+
+Person::Person(const std::string& name, int age){
+    name_=name;
+    age_ = new int (age);
+    *age_ = age;
```

Comment:
This line is unnecessary - ln 19 already sets *age_ to the age

Reply:
Though as a cool tip, you can do this function like this:
`Person::Person(const std::string& name, int age): name_(name), age_(new int(age)){}`